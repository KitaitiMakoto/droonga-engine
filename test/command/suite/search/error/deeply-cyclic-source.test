#@include fixture/user-table.jsons
{
  "type": "search",
  "dataset": "Default",
  "body": {
    "queries": {
      "a": {
        "source": "b"
      },
      "b": {
        "source": "a"
      }
    }
  }
}
