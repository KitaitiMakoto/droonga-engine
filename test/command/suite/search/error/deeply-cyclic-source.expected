{
  "inReplyTo": "request-id",
  "statusCode": 400,
  "type": "search.result",
  "body": {
    "name": "CyclicSource",
    "message": "There is cyclic reference of sources.",
    "detail": {
      "a": {
        "source": "b"
      },
      "b": {
        "source": "a"
      }
    }
  }
}
